src/openforms/js/components/admin/form_design/data/complete-form.js

```diff
@@ -95,6 +95,24 @@ const handleAppointmentForm = draft => {
   draft.formVariables = [];
 };
 
+/**
+ * Options for ZGW registration backend can be empty strings but the serializer does not allow them.
+ * This is the way the regular forms treat options, they don't send data for the field when empty str.
+ */
+const handleZgwRegistrationOptions = draft => {
+  if (draft.form?.registrationBackends) {
+    draft.form.registrationBackends.forEach(backend => {
+      if (backend?.backend === 'zgw-create-zaak') {
+        for (const key in backend?.options) {
+          if (backend.options[key] === '') {
+            delete backend.options[key];
+          }
+        }
+      }
+    });
+  }
+};
+
```

Comment:
What about the case where you have a value specified in one of the fields (form specific configuration), but you want to revert back to using the global configuration? Does it let you empty the field?

Reply:
Yes, basically this is the exact reason I added this here. Without this I get an error from the backend that the field cannot be empty (empty string).